Essential-HighAvailability-changed (#23010)
pingcap/docs: tidb-cloud/serverless-private-link-connection-to-aws-rds.md

> **Note**
>

> > Private Link currently does not support cross-region connectivity. For cross-region connections, it is recommended to use VPC peering to establish a link (TiDB Cloud Premium only feature).

Reviewer: ```suggestion
> Currently, cross-region connectivity is not supported for {{{ .essential }}}. If you need cross-region connections, use TiDB Cloud Premium and establish connectivity through VPC peering.
```
